src/main/java/uk/gov/hmcts/reform/sscs/docmosis/service/DocmosisPdfGenerationService.java

```diff
@@ -15,8 +16,10 @@
 @Slf4j
 public class DocmosisPdfGenerationService implements PdfGenerationService {
 
+    @Value("${service.pdf-service.uri}")
```

Comment:
Please check this ins't going to cause issues with services using this library e.g. notifications

Reply:
checked, it's safe because the service class where that value is defined is not loaded in spring automatically, therefore, the value is not going to be injected.